gtsfm/runner/plot_metrics.py

```diff
@@ -0,0 +1,47 @@
+"""Script to generate a report of metrics with tables and plots 
```

Comment:
Should this file be in runner?

The visualization script for the 3D scene lives in a different folder. I think this script should also live somewhere else, not sure where exactly.

Reply:
how about in `visualization`?

Reply:
Can we put it in `evaluation` itself? I thought all scripts were in runner.